### PR TITLE
Added install instructions to README

### PR DESCRIPTION
**Pull request by ThatOneCalculator**

### CODE REVIEW

**Comment by willeccles** [comment]
I'm hesitant to merge this just because the install instructions are fairly complicated. I see no reason why the install instructions couldn't be something like `sudo install -m755 cpm /usr/bin/cpm` or something. Is there a reason for your complicated instructions? I'm definitely open to suggestions.

**Comment by ThatOneCalculator** [comment]
I wasn't aware of that install command, I'll look into that!

**Comment by ThatOneCalculator** [comment]
@willeccles Ok, I tested and implemented that instruction. Please re-review the new README! :smile:

File: README.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ $ cpm [i|r|l|u|U|s|S|c|h] [pkg]...
 -> h|help    show this message
 ```
 
+## Installation:
```
Comment (willeccles) [inline README.md:22]: Also, remove the colon from this line, like with Usage. In fact, I'd appreciate if you removed all the colons on the header lines, not sure why I put them there in the first place.

**Review by willeccles — CHANGES_REQUESTED** [review]

**Comment by ThatOneCalculator** [comment]
Done!

**Pull request merged**